MAUI/DataForm/ManualDataFormItem.md

```diff
@@ -0,0 +1,209 @@
+                    <dataForm:DataFormGroupItem Name="Address">
+                        <dataForm:DataFormGroupItem.Items>
+                            <dataForm:DataFormMultilineItem FieldName="Street"/>
+                            <dataForm:DataFormTextItem FieldName="State"/>
+                            <dataForm:DataFormTextItem FieldName="ZipCode"/>
+                            <dataForm:DataFormAutoCompleteItem FieldName="Country" ItemsSource = "{Binding CountryNames}"/>
+                        </dataForm:DataFormGroupItem.Items>
+                    </dataForm:DataFormGroupItem>
+                </dataForm:SfDataForm.Items>
+            </dataForm:SfDataForm>
+        </ContentPage.Content>
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+    DataFormViewModel contactInfoViewModel = new DataFormViewModel();
+    dataForm.DataObject = contactInfoViewModel.ContactDetails; 
+    ObservableCollection<DataFormViewItem> items = new ObservableCollection<DataFormViewItem>(); 
+    items.Add(new DataFormTextItem() { FieldName = "Name" }); 
+    items.Add(new DataFormPasswordItem() { FieldName = "Password" }); 
+    items.Add(new DataFormMultilineItem() { FieldName = "Address" }); 
+    DataFormGroupItem groupItem = new DataFormGroupItem();
+    groupItem.Name = "Address";
+    groupItem.Items.Add(new DataFormMultilineItem { FieldName = "Street" });
+    groupItem.Items.Add(new DataFormTextItem { FieldName = "State" });
+    groupItem.Items.Add(new DataFormTextItem { FieldName = "ZipCode" });
+    groupItem.Items.Add(new DataFormAutoCompleteItem { FieldName = "Country", ItemsSource = contactInfoViewModel.CountryNames }); 
+        items.Add(groupItem);
+
+    dataForm.AutoGenerateItems = false; 
+    dataForm.Items = items;
+
+{% endhighlight}   
+{% endtabs %} 
+
+{% endhighlight %}
+{% highlight C# %}
+
+    public class DataFormViewModel
+    {
+        public DataFormViewModel()
+        {
+            this.ContactDetails = new ContactInfo();
+            this.CountryNames = new List<string>
+            {
+                "United States",
+                "United Kingdom",
+                "France",
+                "Belgium",
+                "Germany"
+            };
+        }
+
+        public ContactInfo ContactDetails { get; set; }
+
+        public IList<string> CountryNames { get; set; }
+
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+{% endhighlight %}
+{% highlight C# %}
+
+    public class ContactInfo
+    {
+        public string Name { get; set; }
+
+        public string Password { get; set; }
+
+        public string Street { get; set; }
+
+        public string City { get; set; }
+
+        public string ZipCode { get; set; }
+
+        public string Country { get; set; }
+
+        public string Email { get; set; }
+    }
+
+{% endhighlight %}
+
+## Dynamically add DataForm items
+You can dynamically add the items using `Items` property of `SfDataForm`.
```

Comment:
You can dynamically add the data editor manually by using `Items` property of `SfDataForm`.

Reply:
content updated